how-to-guides/neptune-docker/scripts/training.py

```diff
@@ -0,0 +1,89 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torchvision import datasets, transforms
+import neptune.new as neptune
+
+# Step 1: Initialize Neptune and create new Neptune Run
+run = neptune.init(
+    project="common/pytorch-integration",
+    api_token="ANONYMOUS",
```

Comment:
Let me know if you decided to pass token here and do not work on docker secrets. For me both ways are ok

Reply:
Docker Secrets !!!